x-pack/plugins/maps/ui_tests/tests/full_screen_mode.spec.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, tags } from '@kbn/scout';
+import { test } from '../fixtures';
+
+test.describe('maps full screen mode', { tag: tags.ESS_ONLY } , () => {
```

Comment:
does it mean the behaviour on Serverless is different? Could you share the principal differences?

Reply:
None, and this is far from ready for review.  For now it's just copy-pasta on purpose.
Just to get things running on my local

Reply:
[Changed](https://github.com/elastic/kibana/pull/204607/files#diff-e42d3ef862f5fd5637f38bab2714487e8eb5dc5f802e5fbbc231420aaebe12c2R10) to DA